src/test/java/com/softserveinc/ita/TestsTest.java

```diff
@@ -111,6 +117,12 @@ public void verifyDeletingTest() {
         soft.assertAll();
     }
 
+    @Test(groups = "positive")
+    @Description("Test to verify corectness of the json schema of tests entities")
+    public void verifyTestEntitySchemaRecords () {
```

Comment:
better to create a separate class for schema verifications, and without using of testrunner